docs/preview/features/key-vault/extensions/key-vault-secret-source.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Azure Key Vault secret store"
+layout: default
+---
+
+# Azure Key Vault Secret Store
```

Comment:
We should refer to them as providers similar to .NET's Configuration imo

Reply:
Not sure if I follow, how exactly do you wanna this to change?

Reply:
Changed (I think) all references to child registrations as 'secret source/store' to 'secret provider'.